source/frontends/libretro/game.cpp

```diff
@@ -47,13 +26,11 @@ namespace ra2
     , myAudioSource(AudioSource::UNKNOWN)
     , myKeyboardType(KeyboardType::ASCII)
   {
-    myLoggerContext = std::make_shared<LoggerContext>(true);
+    myLoggerContext = std::make_shared<LoggerContext>(false);
```

Comment:
similar to above, the core should not be writing files unnecessarily. this could be modified to write to the frontend log via `log_cb`, but I felt it was reasonable to just disable it entirely.

Reply:
right now, this is premature. will be addressed once the compilation is fixed.